storybook/src/stories/UseDeepCompareEffect/stories.tsx

```diff
@@ -1,7 +1,7 @@
 import type { Meta, StoryObj } from '@storybook/react'
 import API from './API'
 import API_CODE from './API/API.tsx?raw'
-import { COUNTRY_CODES } from './API/constants'
+import COUNTRY_CODES from './API/countryCodes.json'
```

Comment:
This might be a good case for an `assets` subfolder if we want to be opinionated about how we're using JSON files

```suggestion
import COUNTRY_CODES from './API/assets/countryCodes.json'
```